src/radius/wpabuf.h

```diff
@@ -22,66 +19,20 @@
 #include "utils/allocs.h"
 #include "utils/os.h"
 
-/*
- * Definitions for sparse validation
- * (http://kernel.org/pub/linux/kernel/people/josh/sparse/)
- */
-#ifdef __CHECKER__
-#define __force __attribute__((force))
-#undef __bitwise
-#define __bitwise __attribute__((bitwise))
-#else
-#define __force
-#undef __bitwise
-#define __bitwise
-#endif
-
 #ifndef WPA_BYTE_SWAP_DEFINED
 
-#ifndef __BYTE_ORDER
-#ifndef __LITTLE_ENDIAN
-#ifndef __BIG_ENDIAN
-#define __LITTLE_ENDIAN 1234
-#define __BIG_ENDIAN 4321
-#if defined(sparc)
-#define __BYTE_ORDER __BIG_ENDIAN
-#endif
-#endif /* __BIG_ENDIAN */
-#endif /* __LITTLE_ENDIAN */
-#endif /* __BYTE_ORDER */
-
-#if __BYTE_ORDER == __LITTLE_ENDIAN
-#define le_to_host16(n) ((__force u16)(le16)(n))
-#define host_to_le16(n) ((__force le16)(uint16_t)(n))
-#define be_to_host16(n) bswap_16((__force uint16_t)(uint16_t)(n))
-#define host_to_be16(n) ((__force uint16_t)bswap_16((n)))
-#define le_to_host32(n) ((__force u32)(le32)(n))
-#define host_to_le32(n) ((__force le32)(uint32_t)(n))
-#define be_to_host32(n) bswap_32((__force uint32_t)(be32)(n))
-#define host_to_be32(n) ((__force be32)bswap_32((n)))
-#define le_to_host64(n) ((__force u64)(le64)(n))
-#define host_to_le64(n) ((__force le64)(uint64_t)(n))
-#define be_to_host64(n) bswap_64((__force uint64_t)(be64)(n))
-#define host_to_be64(n) ((__force be64)bswap_64((n)))
-#elif __BYTE_ORDER == __BIG_ENDIAN
-#define le_to_host16(n) bswap_16(n)
-#define host_to_le16(n) bswap_16(n)
-#define be_to_host16(n) (n)
-#define host_to_be16(n) (n)
-#define le_to_host32(n) bswap_32(n)
-#define host_to_le32(n) bswap_32(n)
-#define be_to_host32(n) (n)
-#define host_to_be32(n) (n)
-#define le_to_host64(n) bswap_64(n)
-#define host_to_le64(n) bswap_64(n)
-#define be_to_host64(n) (n)
-#define host_to_be64(n) (n)
-#ifndef WORDS_BIGENDIAN
-#define WORDS_BIGENDIAN
```

Comment:
This `#define` is the same as the one in https://github.com/nqminds/edgesec/blob/2ae010960fb1ad48a38a95ec2e7331285e63f82d/src/radius/md5_internal.c#L66,
however `wpabuf.h` is not `#include`-ed by `md5_internal.c`, so it's safe to remove.

Reply:
As long as the radius test rusn. Should be fine.